### ISSUE TITLE
Introduce TableView

### ISSUE THREAD

**Issue by wateret** [wateret/mengde#111]
It would be nice if we can have a table (list) view.

Requirements for `TableView`

- Can have Header
- Can add rows
- Can have border

**Comment by sjsinju** [comment]
Please assign to me.

**Comment by sjsinju** [comment]
For a naive TableView,
```
int addCol(std::string title)
int addRow(std::initializer_list<View>)
...
```

**Comment by wateret** [comment]
Resolved by #129 Thank you! @sjsinju
